### PR TITLE
[STUB]: `promote_to_draft.yml` GH workflow

### PR DESCRIPTION
**Pull request by sf-dcp**
Need to create a stub of new workflow on `main` in order to be able to test on my other dev brunch.

### CODE REVIEW

File: .github/workflows/promote_to_draft.yml
Comment (fvankrieken) [inline .github/workflows/promote_to_draft.yml:1]: I think without a job it might fail too... Give it a shot though. Sorry!!

Comment (fvankrieken) [reply .github/workflows/promote_to_draft.yml:1]: Oh great I was wrong. I think what I did wrong once was a stub that had a job but no steps, which prevented anything from working even on a dev branch. Or something like that. Carry on!

Comment (fvankrieken) [reply .github/workflows/promote_to_draft.yml:1]: Oh of course now I remember - non-dev branches fail on push (or other dev branches with this commit from main)

https://github.com/NYCPlanning/data-engineering/actions/runs/10098654226

Not terrible, just annoying 🙂

Comment (sf-dcp) [reply .github/workflows/promote_to_draft.yml:1]: oh wow. Good to know!

**Pull request merged**